client/.storybook/main.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
I'll need to make a mental note to make sure none of the `storybook` goes into the `@mdn/yari` npm tarball package.